### PR TITLE
Add OriginTrail Parachain to polkadotjs

### PR DESCRIPTION
**Pull request by kotlarmilos**
Changes:
- add `packages/apps-config/src/ui/logos/nodes/starfleet.png`
- update `packages/apps-config/src/api/spec/index.ts`
- update `packages/apps-config/src/api/spec/starfleet.ts`
- update `packages/apps-config/src/endpoints/testingRelayRococo.ts`
- update `packages/apps-config/src/settings/ethereumChains.ts`
- update `packages/apps-config/src/ui/colors.ts`
- update `packages/apps-config/src/ui/logos/index.ts`
- update `packages/apps/public/locales/en/apps-config.json`

### CODE REVIEW

File: packages/apps-config/src/endpoints/testingRelayRococo.ts [hunk 1]
```diff
@@ -393,6 +393,14 @@ export function createRococo (t: TFunction): EndpointOption {
           SubDAONetwork: 'wss://parachain.subdao.network'
         }
       },
+      {
+        info: 'rococoOriginTrail',
+        paraId: 2024,
+        text: t('rpc.origintrail', 'OriginTrail Parachain', { ns: 'apps-config' }),
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRelayRococo.ts:399]: Can we just move this up - alphabetical based on chain. (With the number of items it is a pain to try and locate if not done).

File: packages/apps-config/src/ui/colors.ts [hunk 2]
```diff
@@ -119,6 +119,7 @@ const nodeRealis = 'linear-gradient(45deg, #E8AAC9 0%, #C4D9E7 50%, #EFD6E0 100%
 const nodeRiochain = '#1A3BB3';
 const nodeSora = '#2D2926';
 const nodeStafi = '#00F3AB';
+const nodeOriginTrail = '#131415';
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:122]: Just move alphabetical as well.

File: packages/apps-config/src/ui/colors.ts [hunk 3]
```diff
@@ -234,6 +235,7 @@ export const chainColors: Record<string, string> = [
   ['Statemine Test', specStatemine],
   ['Statemint', specStatemint],
   ['Statemint Test', specStatemint],
+  ['OriginTrail Parachain', nodeOriginTrail],
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:238]: Some here.

File: packages/apps-config/src/ui/colors.ts [hunk 4]
```diff
@@ -304,6 +306,7 @@ export const nodeColors: Record<string, string> = [
   ['Statemine Collator', specStatemint],
   ['Statemint Collator', specStatemint],
   ['SubDAO Collator', nodeSubDAO],
+  ['OriginTrail Parachain', nodeOriginTrail],
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:309]: Move up.

File: packages/apps-config/src/ui/logos/index.ts [hunk 5]
```diff
@@ -73,6 +73,7 @@ import mybank from './nodes/mybank.png';
 import nodeNFTMart from './nodes/nftmart.png';
 import nodeNodle from './nodes/nodle.svg';
 import oakTestnet from './nodes/oak-testnet.png';
+import nodeOriginTrail from './nodes/origintrail.png';
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:76]: Move up. (Under O)

File: packages/apps-config/src/ui/logos/index.ts [hunk 6]
```diff
@@ -187,6 +188,7 @@ export const chainLogos: Record<string, unknown> = [
   ['Statemine Test', nodeStatemine],
   ['Statemint', nodeStatemine],
   ['Statemint Test', nodeStatemine],
+  ['OriginTrail Parachain', nodeOriginTrail],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:191]: Move up.

File: packages/apps-config/src/ui/logos/index.ts [hunk 7]
```diff
@@ -306,6 +308,7 @@ export const nodeLogos: Record<string, unknown> = [
   ['Statemine Collator', nodeStatemine],
   ['Statemint Collator', nodeStatemine],
   ['SubDAO Collator', nodeSubDAO],
+  ['OriginTrail Parachain', nodeOriginTrail],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:311]: Move up. 

(I should really add a test for these, with the fact that it is arrays the linter does not catch it)

**Review by jacogr — COMMENTED** [review]
Looks good now that it is passing. some small comments to make future maintenance easier.

File: packages/apps-config/src/endpoints/testingRelayRococo.ts (hunk 1 above)
Comment (kotlarmilos) [reply packages/apps-config/src/endpoints/testingRelayRococo.ts]: Good point! Done!

File: packages/apps-config/src/ui/colors.ts (hunk 3 above)
Comment (kotlarmilos) [reply packages/apps-config/src/ui/colors.ts]: Done!

File: packages/apps-config/src/ui/colors.ts (hunk 2 above)
Comment (kotlarmilos) [reply packages/apps-config/src/ui/colors.ts]: Done!

File: packages/apps-config/src/ui/logos/index.ts (hunk 5 above)
Comment (kotlarmilos) [reply packages/apps-config/src/ui/logos/index.ts:76]: I think it is at the right place. Please let me know where to move it if necessary

File: packages/apps-config/src/ui/logos/index.ts (hunk 6 above)
Comment (kotlarmilos) [reply packages/apps-config/src/ui/logos/index.ts]: Done!

File: packages/apps-config/src/ui/logos/index.ts (hunk 7 above)
Comment (kotlarmilos) [reply packages/apps-config/src/ui/logos/index.ts]: Done!

**Review by jacogr — APPROVED** [review]
Thank you.

**Comment by riusricardo** [comment]
Thanks @jacogr and @kotlarmilos

**Pull request merged**